portal-api/src/v1/api-response-types/utilisation-report-pending-corrections-response-body.ts

```diff
@@ -0,0 +1,18 @@
+import { IsoDateTimeStamp, ReportPeriod } from '@ukef/dtfs2-common';
+
+type PendingCorrection = {
+  correctionId: number;
+  facilityId: string;
+  exporter: string;
+  additionalInfo: string;
+};
```

Comment:
libs/common? also being used in `dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/get-utilisation-report-pending-corrections.controller/index.ts`